### ISSUE TITLE
Hooks installation, Easyredmine, redmine 3.2.0, gitolite, git_hosting_plugin, phusion passenger (apache2)

### ISSUE THREAD

**Issue by CaulUinTur** [jbox-web/redmine_git_hosting#570]
I have been trying to install a combination of easyredmine with gitolite and the git hosting plugin on debian 8 but I think I am missing some important step or command or configuration.

The checks in config test report Hooks installed? , Mailer parameters installed? and Mirror keys installed? as red exclamation marks.

After googling a lot I have been trying exec rake commands, configuring the .gitolite.rc file, changing the LOCAL_CODE variable and other steps. Ofcourse one of the first things I tried was following the guide at http://redmine-git-hosting.io/get_started/, as well as looking at the redmine and gitolite documentation.

What am I overlooking or missing here?

![hooks tab](https://cloud.githubusercontent.com/assets/17139086/12943018/d304324c-cfde-11e5-852c-fc5bae16799d.png)
![config test](https://cloud.githubusercontent.com/assets/17139086/12943027/d6a22710-cfde-11e5-8e11-417cca742fb5.png)

**Comment by CaulUinTur** [comment]
My git hosting log gives me the error:
Error while getting Gitolite infos, check your SSH keys (path, permissions) or your Git user.

Also, the ssh info command does not return the info described in the step with RW+ access for the gitolite admin.

I tried checking and changing the config files but sofar it does not seem to fix the error.

If I had to guess I would say I made a mistake in the configuration somewhere, which files will I need to check or edit?
